src/main/java/com/nha/abdm/wrapper/hip/hrp/GatewayCallbackController.java

```diff
@@ -183,6 +193,37 @@ public ResponseEntity<GatewayCallbackResponse> onAddCareContext(
     return new ResponseEntity<>(HttpStatus.ACCEPTED);
   }
 
+  @PostMapping({"/v0.5/consents/hip/notify"})
+  public ResponseEntity<GatewayCallbackResponse> hipNotify(
+      @RequestBody HIPNotifyRequest hipNotifyRequest) {
+    if (hipNotifyRequest != null
```

Comment:
nit: A good to have plugin is the spring boot validation which reduces the boilerplate code. A reference example can be found [here](https://medium.com/@tericcabrel/validate-request-body-and-parameter-in-spring-boot-53ca77f97fe9).

Reply:
Added https://github.com/NHA-ABDM/ABDM-wrapper/issues/50 to take care of this.